src/topology/algebra/order/basic.lean

```diff
@@ -1795,8 +1796,7 @@ section continuous_mul
 lemma mul_tendsto_nhds_zero_right (x : α) :
   tendsto (uncurry ((*) : α → α → α)) (𝓝 0 ×ᶠ 𝓝 x) $ 𝓝 0 :=
 begin
-  have hx : 0 < 2 * (1 + |x|) := (mul_pos (zero_lt_two) $
-    lt_of_lt_of_le zero_lt_one $ le_add_of_le_of_nonneg le_rfl (abs_nonneg x)),
+  have hx : 0 < 2 * (1 + |x|) := by positivity,
   rw ((nhds_basis_zero_abs_sub_lt α).prod $ nhds_basis_abs_sub_lt x).tendsto_iff
      (nhds_basis_zero_abs_sub_lt α),
   refine λ ε ε_pos, ⟨(ε/(2 * (1 + |x|)), 1), ⟨div_pos ε_pos hx, zero_lt_one⟩, _⟩,
```

Comment:
can you also replace `div_pos ε_pos hx` and `zero_lt_one` by calls to `positivity` here?

Reply:
In principle, yes, but for the first one both goal and hypothesis are not syntactically in the right form, so you have to write a type annotation like
```lean
refine λ ε (ε_pos : 0 < ε), ⟨(ε/(2 * (1 + |x|)), 1), ⟨(by positivity : 0 < ε / (2 * (1 + |x|))), zero_lt_one⟩, _⟩,
```
And I don't think it's worth switching just the `zero_lt_one` to a positivity call.